<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_DB</class>
 <widget class="QDialog" name="Show_DB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="Current_data_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>931</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_page_btn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="page_LE">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Page_go_label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Перейти на страницу:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="Page_txt1">
      <property name="text">
       <string>Страница</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="page_number_labl">
      <property name="text">
       <string>1/1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
